component-library/src/app/pages/date-picker-documentation/date-picker-doc-code.component.ts

```diff
@@ -0,0 +1,340 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, Validators } from '@angular/forms';
+import { FormControl } from '@angular/forms';
+import { FormGroup } from '@angular/forms';
+import {
+  ICodeViewerConfig,
+  stringify
+} from '@app/components/code-viewer/code-viewer.component';
+import {
+  slugAnchorType,
+  slugTitleURLConfig
+} from '@app/components/title-slug-url/title-slug-url.component';
+import { SlugifyPipe } from '@app/share/pipe-slugify.pipe';
+import { TranslateService } from '@ngx-translate/core';
+import {
+  ICheckBoxComponentConfig,
+  IDatePickerConfig,
+  IDatePickerErrorMessages,
+  IRadioInputComponentConfig
+} from 'ircc-ds-angular-component-library';
+import { LangSwitchService } from '../../share/lan-switch/lang-switch.service';
+
+@Component({
+  selector: 'app-date-picker-doc-code',
+  templateUrl: './date-picker-doc-code.component.html',
+  styleUrls: ['./date-picker-doc-code.component.scss'],
+  providers: [SlugifyPipe]
+})
+export class DatePickerDocCodeComponent implements OnInit {
+  currentLanguage: string = '';
+  altLangLink = 'datePickerDocCode';
+  form_datePicker = new FormGroup({});
+  errorState = 'None';
+
+  datePickerErrorMessages: IDatePickerErrorMessages = {
+    general: [
+      {
+        key: 'required',
+        errorLOV: 'ACC_DEMO.ERRORS.DOB_REQUIRED'
+      }
+    ]
+  };
+
+  datePickerConfig: IDatePickerConfig = {
+    id: 'datePicker',
+    formGroup: this.form_datePicker,
+    label: 'This is Label',
+    required: true,
+    desc: 'Description line of text',
+    size: 'small',
+    errorMessages: this.datePickerErrorMessages,
+    unknownDateToggle: {
+      dayUnknown: true,
+      monthUnknown: true,
+      yearUnknown: true
+    },
+    disabled: false
+  };
+
+  toggles: IRadioInputComponentConfig[] = [
+    {
+      id: 'size',
+      formGroup: this.form_datePicker,
+      size: 'small',
+      label: 'Size',
+      options: [
+        {
+          text: 'Small'
```

Comment:
I know you haven't finished the interactive demo yet, but don't forget to use translation keys here. similar to this:
![image](https://github.com/IRCC-ca/ds-sdc-dev/assets/114417946/e2ed1e5c-b9a6-46dc-a832-4d37c905ce44)

Reply:
Fixed! Thanks :)

Reply:
We need to discuss this again, we can connect on Monday

Reply:
will have to add a value tag aswell, with the respected value.